tasks/client_keys.yml

```diff
@@ -77,7 +77,7 @@
 - name: Fetch client config
   fetch:
     src: "{{ openvpn_ovpn_dir }}/{{ item }}-{{ inventory_hostname }}.ovpn"
-    dest: "{{ openvpn_fetch_client_configs_dir }}/{{ item }}/{{ inventory_hostname }}{{ openvpn_fetch_client_configs_suffix }}.ovpn"
+    dest: "{{ openvpn_fetch_client_configs_dir }}/{{ item }}-{{ inventory_hostname }}{{ openvpn_fetch_client_configs_suffix }}.ovpn"
```

Comment:
Why is this changed? 
This will put all the configs in a single folder instead of having a directory per client.

I don't really want to change it for everyone using this without justification.

Reply:
I made this change firstly for myself because whole file name contains user name after this - more usable when you sending .ovpn profile file to user after creation. Github automatically applied this commit to same PR, thats why it here. Of course this is not necessary, hope you can decline 1 commit (can you?)

Reply:
Nope, it's not giving me the opportunity to undo individual commits.

Just do a commit over it?